src/api/useTerminologyServiceApi.ts

```diff
@@ -66,24 +68,29 @@ export class TerminologyServiceApi {
     }
   }
 
-  async getQdmValueSetsExpansion(cqmMeasure: CqmMeasure): Promise<ValueSet[]> {
+  async getQdmValueSetsExpansion(
+    cqmMeasure: CqmMeasure,
+    manifestExpansion: ManifestExpansion
+  ): Promise<ValueSet[]> {
     if (!cqmMeasure) {
       return null;
     }
-    const searchCriteria = {
+    const searchCriteria: ValueSetsSearchCriteria = {
       includeDraft: "yes", // always yes for now
+      profile: "",
+      manifestExpansion: manifestExpansion,
       valueSetParams: this.getValueSetsOIDsFromCqmMeasure(
         JSON.parse(JSON.stringify(cqmMeasure))
       ),
-    } as ValueSetsSearchCriteria;
+    };
 
     if (_.isEmpty(searchCriteria.valueSetParams)) {
       return [];
     }
 
     try {
       const response = await axios.put(
-        `${this.baseUrl}/vsac/qdm/value-sets/searches`,
+        `${this.baseUrl}/terminology/fhir/value-sets/expansion`,
```

Comment:
this is returning QDM valueset. I think making it clear in url would be good. we'll have another one for FHIR. ther we can have /fhir in url
```suggestion
        `${this.baseUrl}/terminology/qdm/value-sets/expansion`,
```

Reply:
Yeah I was skeptical on this, since we are using FHIR Terminology Server for QDM valueSets, We can make that change but from API Perspective this is the base for Controller which uses FHIR Terminology Server instead of VSAC. 

https://github.com/MeasureAuthoringTool/terminology-service/pull/66/files#diff-461d2da50ef275981de31a3163645790fc07ef1114c13988eecf815fe4b3547f:~:text=src/main/java/gov/cms/madie/terminology/controller/VsacFhirTerminologyController.java

Reply:
Ok Contrary What is your opinion on using the following format? Eventually we will get away from SVS services ( for the most part ) and only use the new Server. But even this doesn't differentiate if the expansions are from SVS or FHIR Terminology Server. 

/terminology/value-sets/expansion/fhir
/terminology/value-sets/expansion/qdm

Reply:
I think that should work. SVS is not going to be there eventually. just want to make sure by looking at endpoint we get some indication about the response whether it is QDM or FHIR.

Reply:
Made changes